frontend/src/pages/GuardianProfile.jsx

```
@@ -0,0 +1,409 @@
+                      onChange={handleInputChange}
+                      className="p-2 rounded-lg border border-slate-300 text-slate-800 w-full"
+                      placeholder="Company Location"
+                    />
+                  ) : (
+                    <p className="text-sm text-slate-500">{profileData.location}</p>
+                  )}
+                </div>
+              </div> */}
+              
+              <button className="w-full bg-gradient-to-r from-blue-500 to-cyan-500 text-white font-semibold py-3 px-6 rounded-xl flex items-center justify-center gap-3 mt-8 hover:-translate-y-1 hover:shadow-lg hover:shadow-blue-500/30 transition-all">
+                <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)